test/functions/resolution/errors/badWrite.good

```diff
@@ -1,5 +1,5 @@
 badWrite.chpl:1: error: unresolved call 'write("hi", type int(64))'
-$CHPL_HOME/modules/standard/ChapelIO.chpl:837: note: this candidate did not match: write(const args ...?n)
+$CHPL_HOME/modules/standard/ChapelIO.chpl:842: note: this candidate did not match: write(const args ...?n)
 badWrite.chpl:1: note: because actual argument #2 is a type
-$CHPL_HOME/modules/standard/ChapelIO.chpl:837: note: but is passed to non-type varargs formal 'args'
+$CHPL_HOME/modules/standard/ChapelIO.chpl:842: note: but is passed to non-type varargs formal 'args'
```

Comment:
Would we want a prediff for this case and other line number changes so that we don't have to update the line numbers again?  If so, you could use this prediff as a template: test/functions/operatorOverloads/operatorMethods/nestedMethodBad.prediff

Reply:
Yeah, I agree, thanks for the pointer.

Reply:
Since nothing about that prediff was specific to the script, I added it to the `util/test/` directory and symbolically linked to it for these two tests of mine.  I didn't try to search for other locations that could also use it.